packages/fast-components-react-base/src/horizontal-overflow/horizontal-overflow.tsx

```diff
@@ -274,19 +279,18 @@ class HorizontalOverflow extends Foundation<
      * Get the scroll change data
      */
     private getPositionData = (): PositionChange => {
-        const isLtr: boolean = this.getLTR() === Direction.ltr;
         const distanceRemaining: number =
             this.horizontalOverflowItemsRef.current.scrollWidth -
-            this.horizontalOverflowItemsRef.current.scrollLeft;
+            this.getScrollPosition();
 
-        if (this.horizontalOverflowItemsRef.current.scrollLeft === 0) {
-            return { start: isLtr, end: !isLtr };
+        if (this.getScrollPosition() === 0) {
+            return { start: false, end: true };
         } else if (
             distanceRemaining === this.horizontalOverflowItemsRef.current.clientWidth
         ) {
-            return { start: !isLtr, end: isLtr };
+            return { start: true, end: false };
         } else {
-            return { start: false, end: false };
+            return { start: true, end: true };
```

Comment:
Nit: I would find this a little more readable.

```
const scrollPositionIsAtBeginning: boolean = this.getScrollPosition() === 0;
const scrollPositionIsAtEnd: boolean = distanceRemaining === this.horizontalOverflowItemsRef.current.clientWidth;
const scrollPositionIsInMiddle: boolean = !scrollPositionIsAtEnd && !scrollPositionIsAtBeginning;

return {
    start: scrollPositionIsInMiddle
        ? false
        : scrollPositionIsAtBeginning,
    end: scrollPositionIsInMiddle
        ? false
        : scrollPositionIsAtEnd
}
```

Reply:
yep